src/server/zset_family.h

```diff
@@ -66,6 +67,13 @@ class ZSetFamily {
     bool override = false;
   };
 
+  struct ValidateZMPopResult {
```

Comment:
you do not need this struct to be defined in the header file. please move it to the cc file.